.github/workflows/build_lint.yml

```diff
@@ -38,30 +35,27 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
     - uses: actions/cache@v3
       with:
         path: ~/.cache/pip
         key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('Pipfile.lock') }}
     - name: Install dependencies
-      run: |
-        sudo apt-get install libspatialindex-dev
-        pip install .
-        pip install pytest pytest-cov
```

Comment:
Why is explicit `pytest` install no longer needed? Does that mean users always install `pytest` by default?

Reply:
`pyrest` should be installed via `pip install .[dev]`:

https://github.com/projectmesa/mesa-geo/blob/22ab1fe48cdadd70ef8c82b92426869845d86754/setup.cfg#L54-L59